code/datums/supplypacks/robotics.dm

```diff
@@ -0,0 +1,103 @@
+/*
+*	Here is where any supply packs
+*	related to robotics tasks live.
+*/
+
+/datum/supply_packs/robotics
+	group = "Robotics"
+
+/datum/supply_packs/randomised/robotics
+	group = "Robotics"
+	access = access_robotics
+
+/datum/supply_packs/eng/robotics
+	name = "Robotics assembly crate"
+	contains = list(
+			/obj/item/device/assembly/prox_sensor = 3,
+			/obj/item/weapon/storage/toolbox/electrical,
+			/obj/item/device/flash = 4,
+			/obj/item/weapon/cell/high = 2
+			)
+	cost = 10
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "Robotics assembly"
+	access = access_robotics
+
+/datum/supply_packs/robotics/robolimbs_basic
+	name = "Basic robolimb blueprints"
+	contains = list(
+			/obj/item/weapon/disk/limb/morpheus,
+			/obj/item/weapon/disk/limb/xion
+			)
+	cost = 15
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "Robolimb blueprints (basic)"
+	access = access_robotics
+
+/datum/supply_packs/robotics/robolimbs_adv
+	name = "All robolimb blueprints"
+	contains = list(
+	/obj/item/weapon/disk/limb/bishop,
+	/obj/item/weapon/disk/limb/hesphiastos,
+	/obj/item/weapon/disk/limb/morpheus,
+	/obj/item/weapon/disk/limb/veymed,
+	/obj/item/weapon/disk/limb/wardtakahashi,
+	/obj/item/weapon/disk/limb/xion,
+	/obj/item/weapon/disk/limb/zenghu,
+			)
+	cost = 40
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "Robolimb blueprints (adv)"
+	access = access_robotics
+
+/datum/supply_packs/robotics/mecha_ripley
+	name = "Circuit Crate (\"Ripley\" APLU)"
+	contains = list(
+			/obj/item/weapon/book/manual/ripley_build_and_repair,
+			/obj/item/weapon/circuitboard/mecha/ripley/main,
+			/obj/item/weapon/circuitboard/mecha/ripley/peripherals
+			)
+	cost = 30
+	containertype = /obj/structure/closet/crate/secure
+	containername = "APLU \"Ripley\" Circuit Crate"
+	access = access_robotics
+
+/datum/supply_packs/robotics/mecha_odysseus
```

Comment:
I don't really know why we still have these. These were only ever put in Cargo to begin with because a long time ago the circuit printer wasn't a thing. I'd remove these, honestly.